docker/docker-compose.yml

```
@@ -4,7 +4,7 @@ services:
   www:
     image: ictu/quality-time_proxy:${QUALITY_TIME_VERSION}
     ports:
-      - "${PROXY_PORT:-80}:80"
+      - "80:8080"
```

If 8080 is the default proxy port, shouldn't this be:
 
```suggestion
      - "80:${PROXY_PORT:-8080}"
```

---

We don't actually have any logic that uses the `PROXY_PORT` env var. It's possible to build but then we need to change Nginx config and change exposed port in Dockerfile. Also, we won't have dynamic port mappings for a kubernetes deploy via Helm

---

Addressed this by implementing `PROXY_PORT`.